custom_cards/custom_card_irmajavi_speedtest/custom_card_irmajavi_speedtest.yaml

```diff
@@ -73,7 +73,9 @@ custom_card_irmajavi_speedtest:
         type: "custom:button-card"
         tap_action:
           action: "call-service"
-          service: "speedtestdotnet.speedtest"
+          service: "homeassistant.update_entity"
+          service_data:
+            entity_id: "sensor.speedtest_download"
```

Comment:
Hardcoding an entity is not advised. The sensor can be different for each user. 
Also, this is a breaking change to use a sensor instead of the integration and should be reflected in the accompanying documentation.

Reply:
I changed the hardcoded sensor to use the variable and modified the readme with the breaking change that the speedtestdotnet.speedtest service is nog longer used.